enemies.js

```diff
@@ -238,4 +238,322 @@ class PirahnaPlant {
             }
         }
     };
-};
\ No newline at end of file
+};
+//the hammer brother enemy
+class HammerBro
+{
+    constructor(game,x,y,facing)
+    {
+        Object.assign(this, { game, x, y, facing });
+        this.velocity = { x: Math.pow(-1, this.facing)*PARAMS.BITWIDTH, y: 0 }; // pixels per second
+        this.spritesheet = ASSET_MANAGER.getAsset("./sprites/enemies.png");
+        this.animations = [];
+        this.animations.push(new Animator(this.spritesheet, 179, 89, 16, 25, 2, 0.1, 14, false, true));
+        this.animations.push(new Animator(this.spritesheet, 119, 89, 16, 24, 2, 0.1, 14, false, true));
+        this.paused = true;
+        this.dead = false;
+        this.deadCounter = 0;
+        //this.updateBB();
+        this.swap = 60;
+        this.jump = 240;
+        this.jumpType = 1;
+        this.passThrough = 0;
+        this.throw = 30;
+        this.throws = 0;
+        this.interval = 0;
+        this.throwing = 0;
+        this.beserk = 3600;
+        this.paused = true;
+        this.BB = new BoundingBox(this.x, this.y, PARAMS.BLOCKWIDTH, (1 + 7/16) * PARAMS.BLOCKWIDTH);
+    }
+    updateBB() {
+        this.BB = new BoundingBox(this.x, this.y, PARAMS.BLOCKWIDTH, (1 + 7/16) * PARAMS.BLOCKWIDTH);
+    };
+
+    update() {
+        if (this.paused && this.game.camera.x > this.x - PARAMS.CANVAS_WIDTH) 
+        {
+            this.paused = false;
+        }
+        //changes his direction
+        if(this.game.camera.mario.x < this.x)
+        {
+            this.facing = 1;
+        }
+        else
+        {
+            this.facing = 0;
+        }
+        const FALL_ACC = 1800;
+        //adds in his secret mechanic where after 1 minute he chaces mario
```

Comment:
uh, according to the wiki and my memory, this is NOT part of the game.
https://www.mariowiki.com/Hammer_Bro#Super_Mario_Bros.

feel free to leave the code in, but it should not be on by default which this code seems to be (see this.beserk set to 3600 which is definitely more then 0)